content/pt-br/docs/reference/using-api/health-checks.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Kubernetes API health endpoints
+reviewers:
+- logicalhan
+content_type: conceito
+weight: 50
+---
+
+<!-- overview -->
+O Kubernetes {{< glossary_tooltip term_id="kube-apiserver" text="servidor de API" >}} fornece endpoints de API para indicar o status atual do servidor de API. Esta página descreve esses endpoints da API e explica como você pode usá-los.
+
+<!-- body -->
+
+## Endpoints da API para health
+
+O Kubernetes servidor de API fornece 3 endpoints de API (`healthz`, `livez` and `readyz`) para indicar o status atual do servidor de API.
+O endpoint  `healthz` é descontinuado (desde Kubernetes v1.16), e você deve usar algum endpoint mais específico como `livez` e `readyz`.
+O endpoint `livez` pode ser usado com a [flag](/docs/reference/command-line-tools-reference/kube-apiserver) `--livez-grace-period` para especificar a duração de inicialização.
+Para um desligamento elegante, você pode especificar a [flag](/docs/reference/command-line-tools-reference/kube-apiserver) `--shutdown-delay-duration` junto com o endpoint `/readyz`.
+As máquinas que verificam o `healthz`/`livez`/`readyz` do servidor API devem se basear no código de status HTTP.
+Um código de status `200` indica que o servidor API está `healthy`/`live`/`ready`, dependendo do endpoint chamado.
+As opções mais detalhadas mostradas abaixo destinam-se a ser usadas por operadores humanos para depurar seu cluster ou entender o estado do servidor de API.
```

Comment:
Maybe you can find a better option better than `por operadores humanos`, I'll think to suggest something.